src/adagrams.js

```diff
@@ -1,15 +1,135 @@
+const DISTRIBUTION_OF_LETTERS = {
+  "A": 9,
+  "B": 2,
+  "C": 2,
+  "D": 4,
+  "E": 12,
+  "F": 2,
+  "G": 3,
+  "H": 2,
+  "I": 9,
+  "J": 1,
+  "K": 1,
+  "L": 4,
+  "M": 2,
+  "N": 6,
+  "O": 8,
+  "P": 2,
+  "Q": 1,
+  "R": 6,
+  "S": 4,
+  "T": 6,
+  "U": 4,
+  "V": 2,
+  "W": 2,
+  "X": 1,
+  "Y": 2,
+  "Z": 1
+};
+
+
+const SCORE_CHART = {
+  "A": 1,
+  "B": 3,
+  "C": 3,
+  "D": 2,
+  "E": 1,
+  "F": 4,
+  "G": 2,
+  "H": 4,
+  "I": 1,
+  "J": 8,
+  "K": 5,
+  "L": 1,
+  "M": 3,
+  "N": 1,
+  "O": 1,
+  "P": 3,
+  "Q": 10,
+  "R": 1,
+  "S": 1,
+  "T": 1,
+  "U": 1,
+  "V": 4,
+  "W": 4,
+  "X": 8,
+  "Y": 4,
+  "Z": 10
+}
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let lettersDrawn = [];
+  let availableLetters = [];
+
+for (let letter in DISTRIBUTION_OF_LETTERS) {
+  availableLetters.push(letter);
+}
+
+while (lettersDrawn.length <= 9) {
+  let letterChoice = availableLetters[Math.floor(Math.random() * availableLetters.length)];
+  let letterCount = lettersDrawn.filter(letter => letter === letterChoice).length;
```

Comment:
```suggestion
  const letterCount = lettersDrawn.filter(letter => letter === letterChoice).length;
```
Even though the values of `letterChoice` and `letterCount` change every iteration, they only exist within the scope of the current iteration. So, their values aren't being modified, the entire variable is being re-created from scratch with a new value at the start of a new iteration. Because of this, we can declare them as `const` variables.